test/test_macos_archs.py

```diff
@@ -62,12 +81,12 @@ def test_cross_compiled_test(tmp_path, capfd, build_universal2):
 
     actual_wheels = utils.cibuildwheel_run(
         project_dir,
-        add_env={
-            "CIBW_BUILD": "cp310-*" if build_universal2 else "*p310-*",
-            "CIBW_TEST_COMMAND": '''python -c "import platform; print('running tests on ' + platform.machine())"''',
-            "CIBW_ARCHS": "universal2" if build_universal2 else "x86_64 arm64",
-            "CIBW_BUILD_VERBOSITY": "3",
-        },
+        add_env=dict(
```

Comment:
Nit: I prefer `{}`, it's a bit faster (native syntax) and works in all cases (`dict` requires valid identifier keys), and `**` still works.

Reply:
TIL - I didn't know `**` syntax was legal inside `{}`. I've always fallen back to `dict()` when keyword expansion was needed. I'll push a cleanup.